### PR TITLE
Fix editor screenshots menu button's tooltip

### PR DESCRIPTION
**Pull request by WhalesState**

### CODE REVIEW

**Comment by AThousandShips** [comment]
This breaks at least workflow compatibility, I'd argue this requires a proposal to discuss if this is a good idea in the first place

**Comment by WhalesState** [comment]
> This breaks at least workflow compatibility, I'd argue this requires a proposal to discuss if this is a good idea in the first place

We can just save them separately in the editor data folder, the issue is that it's saving to the project data folder, while only running games should save there, not the editor.

**Comment by AThousandShips** [comment]
My bad you didn't mention this was not the intended behavior, based on the tooltips

However it still needs to be discussed and breaks workflows that rely on the position

**Comment by WhalesState** [comment]
Ok, i will just fix the path without making a separate folder for them.

**Comment by AThousandShips** [comment]
The separate folder isn't the point for that part, you still are moving them to an *entirely different path* that's what breaks workflows

**Comment by AThousandShips** [comment]
The more I think about this the more I'm against this change, the current behavior makes it easy to handle screenshots from different projects, the main use of these screenshots are for tutorials, this way you can organize them easily if you're working on different tutorials at once, for example if you spend some time working on the example data etc. and you screenshot each step and you do all the stages (especially if you, for example, record it as a video as well) and then you write up and use the screenshots at the end of the day

Now with these changes the screenshots will be in one grouped folder with no indication which is from which project

I'd say the proper change here is to fix the tooltip, as it was confused in what it said, whereas the location for the screenshot was intended from the start, it also changes a specific detail that has been in the engine since before 4.0, *and* will now differ between this and past versions (this is IMO not a valid change to cherry pick)

**Comment by WhalesState** [comment]
At least we know that such confusion exists.

I have a question, shouldn't we considered this as deprecated in 4.x? since it's only capturing the main Editor window and not the whole screen.

**Comment by AThousandShips** [comment]
> I have a question, shouldn't we considered this as deprecated in 4.x? since it's only capturing the main Editor window and not the whole screen. It can't include opened menus, or other sub-windows, and the shortcut doesn't work if another window is focused ie. popups.

I'd say that's something to also discuss in a proposal

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -7035,7 +7035,7 @@ EditorNode::EditorNode() {
 	ED_SHORTCUT_OVERRIDE("editor/take_screenshot", "macos", KeyModifierMask::META | Key::F12);
 	settings_menu->add_shortcut(ED_GET_SHORTCUT("editor/take_screenshot"), EDITOR_SCREENSHOT);
 
-	settings_menu->set_item_tooltip(-1, TTR("Screenshots are stored in the Editor Data/Settings Folder."));
+	settings_menu->set_item_tooltip(-1, TTR("Screenshots are stored in the Project Data/Settings Folder."));
```
Comment (AThousandShips) [inline editor/editor_node.cpp:7038]:
```suggestion
	settings_menu->set_item_tooltip(-1, TTR("Screenshots are stored in the User Data (\"user://\") Folder."));
```
Or similar, the other name doesn't name a specific path (except possibly `.godot`)

Comment (WhalesState) [reply editor/editor_node.cpp]: Thank you, and I'm sorry for any misconceptions, have a good day ^

**Review by AThousandShips — APPROVED** [review]
LGTM but needs some editor people reviews as well for specifics

File: editor/editor_node.cpp [hunk 2]
```diff
@@ -7035,7 +7035,7 @@ EditorNode::EditorNode() {
 	ED_SHORTCUT_OVERRIDE("editor/take_screenshot", "macos", KeyModifierMask::META | Key::F12);
 	settings_menu->add_shortcut(ED_GET_SHORTCUT("editor/take_screenshot"), EDITOR_SCREENSHOT);
 
-	settings_menu->set_item_tooltip(-1, TTR("Screenshots are stored in the Editor Data/Settings Folder."));
+	settings_menu->set_item_tooltip(-1, TTR("Screenshots are stored in the User Data (\"user://\") Folder."));
```
Comment (akien-mga) [inline editor/editor_node.cpp:7038]:
```suggestion
	settings_menu->set_item_tooltip(-1, TTR("Screenshots are stored in the user data folder (\"user://\")."));
```

**Comment by akien-mga** [comment]
IMO the location where screenshots are saved is unexpected, so we should consider changing it. I'm not sure what's the best location but the project-specific `user://` folder isn't it.

But for now it makes sense to document this better for 4.3, and evaluate changing it for a future release.

**Comment by WhalesState** [comment]
IMO, the best location is in editor data folder, we can create a new folder "Screenshots" and we save the screenshots there like `ProjectName + "_screenshot_"+ Time` this will make it easier to sort them.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**